### PR TITLE
Correct typo for possive voice and verbs infinitive

### PR DESCRIPTION
**Pull request by cikay**

### CODE REVIEW

File: kmr_mg-ud-test.conllu [hunk 1]
```diff
@@ -6659,20 +6659,20 @@
 15	.	.	PUNCT	sent	_	3	punct	_	_
 
 # sent_id = wiki:wikibank.vislcg.txt:54:1468
-# text = Di 647an Iran, Lîbya hat dagir kirin û dagir kirina Anatoliya jî dest pê kir.
+# text = Di 647an Iran, Lîbya hat dagirkirin û dagir kirina Anatoliya jî dest pê kir.
```
Comment (cikay) [inline kmr_mg-ud-test.conllu:6662]: In passive voice "lêkerên hevdudanî" are written adjacent. [See](https://zanistvan.wordpress.com/wp-content/uploads/2020/12/pirtuken-zimane-kurdi-1.pdf) page 80. Page number written bottom of page.

File: kmr_mg-ud-test.conllu [hunk 2]
```diff
@@ -6659,20 +6659,20 @@
...
 1	Di	di	ADP	pr	AdpType=Prep	2	case	_	_
 2	647an	647	NUM	num	Case=Acc|Number=Plur|NumType=Card	7	nmod	_	_
 3	Iran	îran	PROPN	np	Case=Nom|Definite=Def|Gender=Fem|Number=Sing	7	nsubj	_	SpaceAfter=No
 4	,	,	PUNCT	cm	_	5	punct	_	_
 5	Lîbya	Lîbya	PROPN	np	Case=Nom|Gender=Masc|Number=Sing	3	conj	_	_
 6	hat	hatin	AUX	vblex	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	7	aux	_	_
-7	dagir kirin	dagirkirin	VERB	vblex	VerbForm=Inf	0	root	_	_
+7	dagirkirin	dagir kirin	VERB	vblex	VerbForm=Inf	0	root	_	_
```
Comment (cikay) [inline kmr_mg-ud-test.conllu:6669]: The verb infinitive is "dagir kirin". [See](https://ku.wiktionary.org/wiki/dagir_kirin)

File: kmr_mg-ud-test.conllu [hunk 3]
```diff
@@ -9442,8 +9442,7 @@
 6	gelek	gelek	DET	det	Number=Plur|PronType=Ind	7	det	_	_
 7	Spor	spor	NOUN	n	Case=Nom|Definite=Def|Gender=Fem|Number=Plur	10	nsubj	_	_
 8	hatin	hatin	AUX	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	10	aux	_	_
-9	pêk	pêk	ADV	adv	_	10	compound:lvc	_	_
-10	anîn	anîn	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No
+10	pêkanîn	pêkanîn	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No
```
Comment (cikay) [inline kmr_mg-ud-test.conllu:9445]: The same as https://github.com/UniversalDependencies/UD_Kurmanji-MG/pull/3#discussion_r1667417983

File: kmr_mg-ud-test.conllu [hunk 4]
```diff
@@ -6659,20 +6659,20 @@
...
 6	hat	hatin	AUX	vblex	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	7	aux	_	_
-7	dagir kirin	dagirkirin	VERB	vblex	VerbForm=Inf	0	root	_	_
+7	dagirkirin	dagir kirin	VERB	vblex	VerbForm=Inf	0	root	_	_
 8	û	û	CCONJ	cnjcoo	_	13	cc	_	_
 9	dagir	dagir	NOUN	n	Case=Nom|Definite=Def|Gender=Fem|Number=Sing	13	nsubj	_	_
 10	kirina	kirin	NOUN	n	Case=Con|Definite=Def|Gender=Fem|Number=Sing	9	compound	_	_
 11	Anatoliya	Anatoliya	PROPN	np	Case=Nom|Definite=Def|Gender=Fem|Number=Sing	10	nmod:poss	_	_
 12	jî	jî	PART	emph	_	13	advmod	_	_
-13	dest pê kir	destpêkirin	VERB	vblex	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	7	conj	_	SpaceAfter=No
+13	dest pê kir	dest pê kirin	VERB	vblex	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	7	conj	_	SpaceAfter=No
```
Comment (cikay) [inline kmr_mg-ud-test.conllu:6675]: The same as https://github.com/UniversalDependencies/UD_Kurmanji-MG/pull/3#discussion_r1667417983

**Comment by dan-zeman** [comment]
Thank you for the suggestions. As indicated in the CONTRIBUTING file, we cannot accept PRs against the master branch, so I am going to close this PR now. (The correct way would be to make dev the target branch, and make sure that the history is based on dev too, not on master.)

As per the contents itself: See [https://universaldependencies.org/u/overview/typos.html](https://universaldependencies.org/u/overview/typos.html). The policy in UD is that typos are not corrected in the text. Instead, the misspelled text is annotated and where appropriate, `Typo=Yes` is added in FEATS and `CorrectForm=...` in MISC.

This seems to be indeed a typo in [Wikipedia](https://ku.wikipedia.org/wiki/Werzi%C5%9F).

**Pull request closed without merging**